cpmpy/expressions/python_builtins.py

```diff
@@ -97,6 +98,6 @@ def sum(iterable):
         otherwise, makes a sum Operator directly on `iterable`
     """
     iterable = list(iterable) # Fix generator polling
-    if not any(isinstance(elem, Expression) for elem in iterable):
+    if is_false_cst(any(isinstance(elem, Expression) for elem in iterable)):
```

Comment:
I think this is what is breaking the tests on my machine
(does this mean that the examples are not run on the github CI? nor on your computer?)